### DISCUSSION TITLE
마이크 볼륨 조절이 안됩니다.

### DISCUSSION THREAD

**Discussion by IronKoh** [metamong-friends/metamong-friends.github.io#45] [제보하기]
![image](https://github.com/metamong-friends/metamong-friends.github.io/assets/169414893/53c37b81-2e0a-4fd9-9fe1-7e99b6ef1e3e)

설정에서 마이크 볼륨 조절이 안됩니다.
저 뿐만 아니라 다른 분들도 안된다고 하네요.

**Comment by ACT-TJ** [comment]
안녕하세요!

결함으로 보이는데 VDI 동시 사용 환경으로 인한 부분인지.. 
저희 업무상황에서 재현이 잘 되지않아 확인중에 있습니다.
확인되는 데로 조치해서 공유 드리도록 하겠습니다.

감사합니다.

**Comment by ACT-TJ** [comment]
잠시 발생을 해었던 이슈같은데 현재 ProjectView 팀과 저희 모두 재현이 되지 않는 상황으로
해당 이슈는 Close 를 하도록 하겠습니다. 추가로 문제발생 시 다시 확인해 보도록 하겠습니다~!

**Comment by shlee303** [comment]
@IronKoh 마이크 off 상태일때 볼륨바 조절이 안되는 케이스였는데, off여도 작동하도록 조치하였습니다. 제보 감사합니다!
